Gemfile

```diff
@@ -6,11 +6,13 @@ git_source(:github) do |repo_name|
 end
 
 gem 'activeadmin'
+gem 'activeadmin_addons'
```

Comment:
I would recommend locking version of the gems you use. At least you can use `~>` although I prefer specific version.